README.md

```diff
@@ -16,6 +16,8 @@ Please see the extended documentation [here](/docs/index.md)
 
 ### Local Playground
 
+If you need quickly spin-up local environment including all prerequisites, follow [How to run OhMyGLB document](./docs/how-to-run-ohmyglb.md)
+
 ####  Deploy local cluster
```

Comment:
`Local Playground` next to `Deploy local cluster` seems to be a little bit confusing. It also relates to a suggestion to move existing single cluster setup doc out of the README, see below